e2e/__tests__/crawlSymlinks.test.ts

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import {tmpdir} from 'os';
+import * as path from 'path';
```

Comment:
sorry about the lint error, I changed this in master. eslint should autofix it, tho

Reply:
No worries! Fixed